spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/PropertySourceUtils.java

```diff
@@ -54,20 +59,15 @@
 			return properties;
 		}
 		catch (IOException e) {
-			throw new IllegalArgumentException();
```

Comment:
should be obvious that a `UncheckedIOException(e)` will provide a far better insight of what happened vs a `IllegalArgumentException()`.